export/platform/gc_design_system/airtable/main.py

```diff
@@ -0,0 +1,87 @@
+import json
+import boto3
+import os
+import requests
+
+# Environment variables
+AIRTABLE_API_KEY_PARAMETER_NAME = os.environ.get("AIRTABLE_API_KEY_PARAMETER_NAME")
+S3_BUCKET_NAME = os.environ.get("S3_BUCKET_NAME")
+S3_OBJECT_PREFIX = os.environ.get("S3_OBJECT_PREFIX")
+
+# Airtable configuration (hardcoded like Freshdesk domain)
+AIRTABLE_BASE_ID = "appaMppKljeU8zJE1"
+AIRTABLE_TABLE_NAME = "tbllQrJwtozOb0Ziv"
+
+
+def get_airtable_api_key():
+    """Retrieve Airtable API key from SSM Parameter Store."""
+    try:
+        ssm = boto3.client("ssm")  # Create client when needed
+        response = ssm.get_parameter(
+            Name=AIRTABLE_API_KEY_PARAMETER_NAME, WithDecryption=True
+        )
+        return response["Parameter"]["Value"]
+    except Exception as e:
+        raise Exception(f"Failed to retrieve API key from SSM: {str(e)}")
+
+
+def fetch_all_records():
+    """Fetch all records from Airtable with pagination support."""
+    api_key = get_airtable_api_key()
+    base_url = f"https://api.airtable.com/v0/{AIRTABLE_BASE_ID}/{AIRTABLE_TABLE_NAME}"
+    all_records = []
+    offset = None
+
+    headers = {"Authorization": f"Bearer {api_key}"}
+
+    while True:
```

Comment:
Do you know if the Airtable API has throttling limits?  I hit that on other ones and had to add a bit of time between subsequent requests.